api/environments/local

```diff
@@ -21,3 +21,4 @@ DATABASE_TIMEOUT=5
 RUNNING_IN_CLOUD=False
 SERVICE=api
 
+S3_TRANSCRIPTOME_INDEX_BUCKET_NAME=NO_BUCKET
```

Comment:
I suggest making this None so we can use it programmatically if need be.

Reply:
Our current implementation of get_env_variable fails whenever a falsey value is set as the environment variable. I initially had it set as an empty string but then the program would error. Is there a circumstance where this behaviour is needed/expected? If not, should I just rewrite the get_env_variable function?